web_src/js/index.js

```diff
@@ -319,7 +330,76 @@ function uploadFile(file, callback) {
 }
 
 function reload() {
-  window.location.reload();
+  if ($('#current-card-details').length > 0) {
+    reloadIssuesActions();
+  } else {
+    window.location.reload();
+  }
+}
```

Comment:
piggy-backing the page page reload function just doesn't seem right, do this somewhere else.